suite-native/token-definitions/src/tokenDefinitionsMiddleware.ts

```diff
@@ -0,0 +1,69 @@
+import { createMiddlewareWithExtraDeps } from '@suite-common/redux-utils';
+import { getNetworkFeatures, isEthereumBasedNetwork, networks } from '@suite-common/wallet-config';
+import {
+    accountsActions,
+    getTokenDefinitionThunk,
+    selectSpecificTokenDefinition,
+} from '@suite-common/wallet-core';
+
+export const prepareTokenDefinitionsMiddleware = createMiddlewareWithExtraDeps(
+    (action, { dispatch, next, getState }) => {
+        next(action);
+
+        if (
+            accountsActions.createAccount.match(action) ||
+            accountsActions.updateAccount.match(action)
```

Comment:
You can use `isAnyOf` https://redux-toolkit.js.org/api/matching-utilities#isanyof

Reply:
fixed by 6cf5d4c8fc